demos/common/python/requirements.txt

```diff
@@ -1,2 +1,3 @@
 numpy==1.19.5
 opencv-python==4.5.*
+scipy==1.7.1
```

Comment:
@akorobeinikov  could you please explain why it is required for model api?

Reply:
ok, I found https://github.com/openvinotoolkit/open_model_zoo/blob/develop/demos/common/python/openvino/model_zoo/model_api/models/hpe_associative_embedding.py#L18

Reply:
fyi @eizamaliev @anzhella-pankratova

Reply:
there is common dependency resolving policy for all OV components, please use the same scipy version, which used in 
https://github.com/openvinotoolkit/open_model_zoo/blob/develop/tools/accuracy_checker/requirements.in#L25
and https://github.com/openvinotoolkit/openvino/blob/master/tools/pot/setup.py#L44